eurocodedesign/geometry/steelsection/manager.py

```diff
@@ -0,0 +1,168 @@
+""" 
+module for loading and managing the steel section database
+"""
+
+import os
+import pandas as pd
+
+from pathlib import Path
+from eurocodedesign.geometry.steelsection.steelsections import (
+    SteelSection,
+    RolledISection,
+    CircHollowSection,
+    RectHollowSection,
+    SquareHollowSection,
+)
+
+SECTION_DATA = {
+    "HEA": {"filename": "hea_en10365_2017.csv", "section_class": RolledISection},
+    "HEB": {"filename": "heb_en10365_2017.csv", "section_class": RolledISection},
+    "HEM": {"filename": "hem_en10365_2017.csv", "section_class": RolledISection},
+    "IPE": {"filename": "ipe_en10365_2017.csv", "section_class": RolledISection},
+    "UB": {"filename": "ub_en10365_2017.csv", "section_class": RolledISection},
+    "UC": {"filename": "uc_en10365_2017.csv", "section_class": RolledISection},
+    "CHS": {
+        "filename": "chs_en10219_en10210_2006.csv",
+        "section_class": CircHollowSection,
+    },
+    "SHS": {
+        "filename": "shs_en10219_en10210_2006.csv",
+        "section_class": SquareHollowSection,
+    },
+    "RHS": {
+        "filename": "rhs_en10219_en10210_2006.csv",
+        "section_class": RectHollowSection,
+    },
+}
+
+
+def get_data_path() -> Path:
+    """returns the path to where the csv data files are stored"""
+    return Path(os.path.dirname(os.path.realpath(__file__))) / "_data"
+
+
+def is_valid_type(section_name: str) -> bool:
+    """checks that the section type provided is valid
+
+    A section type is valid if a csv datafile exists for the section type
+    provided by section_name, e.g. "IPE", "HEA", "CHS", etc.
+
+    Args:
+        section_name (str): name of the section section e.g. "IPE100"
+    Returns:
+        bool: True if the section type is valid, False otherwise
+    """
+    for section_type in SECTION_DATA.keys():
```

Comment:
I think you could improve the performance and omit the loop by checking "if section_name in _SECTION_DATA.keys()"

Reply:
I don't think this would work, because the keys of the `SECTION_DATA` are the section types `"IPE", "HEA"`, ... etc. but the section name is for example `"IPE100"`.